### ISSUE TITLE
Issue with icon compression

### ISSUE THREAD

**Issue by silouanwright** [figus-cli/figus#13]
I have an icon in Figma that is not getting compressed into svg properly. 

- I and another designer have played around with it quite a bit, to no avail
- It generally does not budge no matter how we arrange the paths.

Here is the icon, as an exported SVG from Figma

![trophy@1x](https://user-images.githubusercontent.com/708820/208991919-68d1ce70-bca2-4f5b-a557-48b36adfcebe.svg)

And there is the SVG that Figus outputs

<img width="193" alt="ScreenShot 2022-12-21 at 13 56 18@2x" src="https://user-images.githubusercontent.com/708820/208992206-b796d990-a8c9-4b35-aa98-21469072a641.png">

Most of the icons that I have put in seem to work (though some require a lot of tweaking). I'm not sure if the issue with this one is due to the way it's put together or what, or if the SVGO compression options should be optionally exposed for users to tweak. Any assistance here would be appreciated.

**Comment by omridevk** [comment]
yes I was indeed thinking of exposing the svgo config. hopefully i'll get to do it soon. can you check if you get the same behvior with the svgo default?
